witnet/my-wit-wallet
# feat: ability clear database for e2e test

- add `testingDeleteStorage` to the globals file
- add defaults to .env example file
- check if we need to delete the database before the initial screen loads
